src/pages/students.js

```diff
@@ -95,15 +98,15 @@ const StudentsPage = ({ location }) => {
                   </h3>
                   <p className={styles.stepDetail}>
                     The Student Edition is offered through the{' '}
-                    <a href="https://education.GitHub.com/benefits">
+                    <a href="https://education.github.com/?utm_source=GitHub-NewRelic">
```

Comment:
Why not here? https://education.github.com/pack

Reply:
This is the link I was asked to use by Github